packages/client/src/views/GrantActivity.vue

```diff
@@ -0,0 +1,167 @@
+<template>
+  <b-card
+    header-bg-variant="white"
+    footer-bg-variant="white"
+  >
+    <template #header>
+      <h3 class="m-2">
+        Grant Activity
+      </h3>
+    </template>
+    <div>
+      <div class="feature-text">
+        Stay up to date with this grant.
+        <b-icon
+          v-b-tooltip
+          class="ml-2"
+          title="Follow this grant to receive an email notification when others follow or leave a note."
+          icon="info-circle-fill"
+        />
+      </div>
+      <b-button
+        block
+        size="lg"
+        :variant="followBtnVariant"
+        class="mb-4"
+        data-follow-btn
+        :disabled="loadingFollowState"
+        @click="toggleFollowState"
+      >
+        <span class="h4">
+          <b-icon
+            icon="check-circle-fill"
+            class="mr-2"
+          />
+        </span>
+        <span class="h5">
+          {{ followBtnLabel }}
+        </span>
+      </b-button>
+      <div>
+        <span
+          v-if="showFollowSummary"
+          data-follow-summary
+        >{{ followSummaryText }}</span>
+        <span
+          v-if="showFollowSummary && showNotesSummary"
+          class="mx-1"
+        >&bull;</span>
+        <span v-if="showNotesSummary">{{ notesSummaryText }}</span>
+      </div>
+    </div>
+
+    <template #footer>
+      <!-- Feed -->
+    </template>
+  </b-card>
+</template>
+
+<script>
+import { mapActions, mapGetters } from 'vuex';
+
+export default {
+  components: {},
+  data() {
+    return {
+      userIsFollowing: false,
+      loadingFollowState: false,
+      followers: [],
+      notes: [],
+    };
+  },
+  computed: {
+    ...mapGetters({
+      loggedInUser: 'users/loggedInUser',
+      currentGrant: 'grants/currentGrant',
+    }),
+    followBtnLabel() {
+      return this.userIsFollowing ? 'Following' : 'Follow';
+    },
+    followBtnVariant() {
+      return this.userIsFollowing ? 'success' : 'primary';
+    },
+    followSummaryText() {
+      const userIsFollower = this.followers.some((follower) => follower.user.id === this.loggedInUser?.id);
```

Comment:
I wonder if this should instead be based on `this.userIsFollowing` – currently, the `userIsFollower` constant will be `false` if the current user is (for example) the first of 52 total followers, or any other scenario where the current user is a follower but isn't represented within the first page of the result set requested by `fetchFollowersState()`.

Reply:
yes this is better - updated